conf/DEV.public.conf

```diff
@@ -81,4 +81,15 @@ contexts {
     }
 }
 
-ophan.endpoint = "http://localhost:9000"
+ophan {
+
+
+    default = TEST
+    testing = TEST
```

Comment:
Maybe we could name this so it's absolutely clear what it does? `whenTestUserLoggedIn` or something like that?

Reply:
I agree, but there's already a precedent in the config file which I don't really want to change in this pull request e.g. it would mean updating the PROD and DEV public and private configs.